src/commands/ls_remote.rs

```diff
@@ -1,20 +1,78 @@
 use crate::config::FnmConfig;
 use crate::remote_node_index;
+use crate::user_version::UserVersion;
+
+use colored::Colorize;
 use thiserror::Error;
 
 #[derive(clap::Parser, Debug)]
-pub struct LsRemote {}
+pub struct LsRemote {
+    /// Filter with SemVer
+    #[arg(long)]
+    filter: Option<UserVersion>,
+
+    /// Only show latest LTS versions
```

Comment:
I'm not sure how to approach the help output, but would this be worthwhile?

```suggestion
    /// Show only LTS versions (all by default, unless provided an LTS codename)
```

```suggestion
    /// Show only LTS versions (optionally filter by LTS codename)
```